documentation/common/version-dependent-attrs.adoc

```diff
@@ -1,3 +1,3 @@
 // Generated by documentation/version-dependent-attrs.sh during the build
 // DO NOT EDIT BY HAND
-:DefaultKafkaVersion: 2.1.0
+:DefaultKafkaVersion: 2.1.1
```

Comment:
Are you sure it's OK to edit this file manually?

Reply:
I run `make all` command, which edited this value.